get-config.js

```diff
@@ -7,7 +7,7 @@ function readConfig() {
 			'This file should exist as `config.js` inside the project root directory.\n' +
 			'Please consult the project README.md for further information.\n'
 		);
-		process.exit( 0 );
+		process.exit( 1 );
```

Comment:
My linter doesn't like this:

```
⚠ at line 10 position 4:  Don't use process.exit(); throw an error instead. (no-process-exit)
```
http://eslint.org/docs/rules/no-process-exit.html